tools/release-checklist

```diff
@@ -134,7 +134,7 @@ git push --tags
 
 ## 5-bis: Alternative for branching
 
-This needs to be more tested, tread with care.
+This needs to be more tested, treat with care.
```

Comment:
This is not a typo.

Reply:
OK, corrected.

Reply:
I would have thought "tread with care" is more used (though the word "tread" isn't). "Tread with care" means "be careful in this context, this might be dangerous". "Treat with care" I would more apply to a baby than to code (treat it with care, love and diapers).

Reply:
FWIW, the expression is "tread carefully" although it's more idiomatic to use it when there's a risk of upsetting someone, rather than making a mistake. Here perhaps "Proceed with caution" might be more idiomatic, but this completely misses the point that the document is for the benefit of those who wrote it!